utilities/src/main/java/io/syndesis/qe/bdd/validation/OpenshiftValidationSteps.java

```diff
@@ -93,6 +94,11 @@ public void deployIRCServer() {
         IrcTemplate.deploy();
     }
 
+    @Given("^deploy OData server$")
+    public void deployODataServer() {
+        WildFlyTemplate.deploy("https://github.com/jsafarik/olingo-sample-service.git", "odata");
```

Comment:
Can this be changed so that the template deploys a subdirectory of this (i.e. syndesis-qe) repository? Not sure it's possible with the wildfly template, but I think it would be better to have the code in one repo.

@avano your thoughts?

Reply:
it would be better but I think it won't be possible as it uses the whole repo in the s2i build and I think it is not possible to alter the s2i build to build a subdir of the repo only.. but I may be completely wrong because I never did that :D

Reply:
we can tweak the s2i image but that may be an overkill for one test. or we can move the repository to syndesisio organization

Reply:
Adding it to syndesis organization seems like the best option then, do you have the rights to do that?

Reply:
I'm open for anything.
The odata service will still need refactoring and there will have to be implemented more stuff (like ETag support once the odata connector supports it).

Reply:
@asmigala @jsafarik I'm now the owner of syndesisio org :smiling_imp: so @jsafarik if you give me some uber permissions to your repo I'll transfer it in the org

Reply:
moved to https://github.com/syndesisio/syndesis-qe-olingo-sample-service